engine/jobmaster/dm/api_test.go

```diff
@@ -241,7 +244,9 @@ func TestQueryStatusAPI(t *testing.T) {
 					"completedTables": 1,
 					"finishedBytes": 100,
 					"finishedRows": 10,
-					"estimateTotalRows": 1000
+					"estimateTotalRows": 1000,
+					"bps": 1000,
+					"progress": "20.00 %"
```

Comment:
20.00 % -> 20.00% ?

Reply:
20.00 % is the correct ref https://github.com/pingcap/tidb/blob/64ca5d48f7dcef83caf9ef23e3c5478493243689/dumpling/export/status.go#L79